traffic_portal/app/src/common/modules/form/cacheGroup/FormCacheGroupController.js

```diff
@@ -95,7 +95,7 @@ var FormCacheGroupController = function(cacheGroup, types, cacheGroups, $scope,
         }
     };
 
-    $scope.fallbackSelected = '';
+    $scope.fallbackSelected = null;
```

Comment:
do these two changes fix a different bug?

Reply:
yeah, i snuck them in there :)

https://github.com/apache/trafficcontrol/issues/3533#issuecomment-562677120